doc/dev_guide/internals/box_protocol.rst

```diff
@@ -1169,6 +1214,73 @@ To see how Tarantool handles this, look at
 `net_box.c <https://github.com/tarantool/tarantool/blob/master/src/box/lua/net_box.c>`_
 function ``netbox_encode_auth``.
 
+..  _box_protocol-streams:
+
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Binary protocol -- streams
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The streams feature, which was added in Tarantool version
+:tarantool-release:`2.10.0-beta1`, allows two things:
+sequential processing and interleaving.
+
+Sequential processing:
+With streams there is a guarantee that the server instance will not
+handle the next request in a stream until it has completed the previous one.
+
+Interleaving:
+A series of requests can include
+"insert for stream #1", "insert for stream #2", "delete for stream #2", "delete
+for stream #1", and then a "commit for stream #1" which the server instance will
+interpret as a request to handle only the two inserts for stream #1.
+
```

Comment:
I don't understand here: you write "insert for stream #1", "insert for stream #2", "delete for stream #2", "delete
for stream #1", and then write "server instance will interpret as a request to handle only the two inserts for stream #1."
There was no two inserts for stream #1. Also it's incorrect. All requests in stream #2 will be processed. If you use stream without transactions (not start transaction using stream:begin()) all requests in stream processed sequently, but without guarantees that transactions give.

Reply:
I see that my interleaving example was bad. I changed it so that it mentions only the requests, and avoided predicting what the server will do.

Reply:
@EvgenyMekhanik is the new version correct? Can we merge & publish this PR?